app/templates/services/edit_submission_section.html

```diff
@@ -1,35 +1,42 @@
 {% extends "services/_base_edit_section_page.html" %}
 
 {% block breadcrumb %}
-  {%
-    with items = [
+  {% with items = [
       {
-        "link": "/",
-        "label": "Digital Marketplace"
+        "text": "Digital Marketplace",
+        "href": "/"
       },
       {
-        "link": url_for(".dashboard"),
-        "label": "Your account"
+        "text": "Your account",
+        "href": url_for('.dashboard')
       },
       {
-        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
-        "label": "Apply to " + framework.name
+        "text": "Apply to " + framework.name,
+        "href": url_for(".framework_dashboard", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_lots", framework_slug=framework.slug),
-        "label": "Services"
+        "text": lot.name,
+        "href": url_for(".framework_submission_lots", framework_slug=framework.slug)
       },
       {
-        "link": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot),
-        "label": service_data.lotName
-      },
-      {
-        "link": url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=service_data.lot, service_id=service_id),
-        "label": service_data.get('serviceName', service_data['lotName'])
-      } if not one_service_limit else {}
+        "text": "Services",
+        "href": url_for(".framework_submission_services", framework_slug=framework.slug, lot_slug=service_data.lot)
+      }
```

Comment:
When I visit http://localhost/suppliers/frameworks/digital-outcomes-and-specialists-4/submissions/digital-outcomes/150182/edit/service-essentials?hidden_text=Service+essentials

![image](https://user-images.githubusercontent.com/503614/71191127-bf106300-227d-11ea-93fb-07c78dd704f2.png)

I think I'm expecting the "Services" and "Digital outcomes" breadcrumbs to be swapped?